microcosm_flask/context.py

```diff
@@ -1,32 +1,22 @@
 from flask import request
-from microcosm.api import defaults
 
 
-X_REQUEST = "X-Request"
+# This is the total set of Globality-proprietary headers
```

Comment:
Are headers actually proprietary?

Reply:
`proprietary` is the wrong word, but these are headers `Globality` and consequently `microcosm` have opinions about. The `X-` part is a HTTP standard, but `X-Request` and `X-Client` are things we made up.

Reply:
Also adding: https://globality.atlassian.net/browse/GLOB-32485?oldIssueView=true for a more hollistic look at context in microcosm.